codalab/worker/main.py

```diff
@@ -110,6 +110,12 @@ def parse_args():
         action='store_true',
         help='If specified the worker quits if it finds itself with no jobs after a checkin',
     )
+    parser.add_argument(
+        '--exit-number-jobs',
```

Comment:
In general, use `run` rather than `job`.  I'd suggest `--exit-after-num-runs`

Reply:
Sure. Just updated!